lib/Bio/EnsEMBL/DataCheck/Checks/CheckChar.pm

```diff
@@ -0,0 +1,85 @@
+=head1 LICENSE
+
+Copyright [2018] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::CheckChar;
```

Comment:
In general, I think it's good to have datachecks with the same name as healthchecks, to help work out equivalencies. But here I'm not sure if that's outweighed by the usefulness of changing to a more meaningful name, e.g. PhenotypeDescriptions; CheckChar is both inaccurate and very generic.

Reply:
Well, no worries. Changed name to PehnotypeDescriptions